core/corehttp/gateway_test.go

```diff
@@ -153,6 +153,12 @@ func TestGatewayGet(t *testing.T) {
 	ns["/ipns/double.example.com"] = path.FromString("/ipns/working.example.com")
 	ns["/ipns/triple.example.com"] = path.FromString("/ipns/double.example.com")
 	ns["/ipns/broken.example.com"] = path.FromString("/ipns/" + k)
+	// We picked .man because:
+	// 1. It's a valid TLD.
+	// 2. Go treats it as the file extension for "man" files (even though
+	//    nobody actually *uses* this extension, AFAIK).
+	// 3. Go accepts "fnord" (the test value) as a valid man file.
```

Comment:
???

Did you mean go does not accept "fnord" as a valid man file?

Reply:
It accepts it as a valid man file *if* the extension is ".man". However, it looks like that part isn't important. I thought go looked at *both* the content *and* the extension but that doesn't appear to be the case (https://github.com/ipfs/go-ipfs/pull/5564#discussion_r223095520).

I don't know how I can make these tests platform-independent. At least they appear to pass on Linux now...